test/include/nntrainer_test_util.h

```diff
@@ -131,6 +131,20 @@ class ScopedIni {
     }                                          \
   } while (0)
 
+#define GEN_TEST_INPUT_B(input, equation_i_j_k_l) \
+  do {                                            \
+    for (int i = 0; i < batch; ++i) {             \
+      for (int j = 0; j < channel; ++j) {         \
+        for (int k = 0; k < height_b; ++k) {      \
+          for (int l = 0; l < width_b; ++l) {     \
+            float val = equation_i_j_k_l;         \
```

Comment:
```suggestion
            float val = (equation_i_j_k_l);         \
```

Reply:
what difference would this make? just curious
will do it anyway